src/backend/distributed/commands/multi_copy.c

```diff
@@ -740,7 +779,7 @@ OpenCopyTransactions(CopyStmt *copyStatement, ShardConnections *shardConnections
 	List *connectionList = NULL;
 
 	MemoryContext localContext = AllocSetContextCreate(CurrentMemoryContext,
-													   "OpenCopyTransactions",
+													   "Open Copy Transactions",
```

Comment:
I memorized this line
TaskTrackerContext = AllocSetContextCreate(TopMemoryContext, "Task Tracker",

but it seems that without spaces are more common. I'm taking back the change.